nemo/collections/nlp/modules/common/megatron/fused_bias_dropout_add.py

```diff
@@ -60,7 +59,6 @@ def bias_dropout_add_fused_train(x, bias, residual, prob):
 def bias_dropout_add_fused_inference_(
     x: torch.Tensor, bias: torch.Tensor, residual: torch.Tensor, prob: float
 ) -> torch.Tensor:
-    # type: (Tensor, Tensor, Tensor, float) -> Tensor
```

Comment:
Revert needed for tracing

Reply:
Ok, reverted

Why duplicated types required? Is it Pytorch bug? (actually, Pycharm complains about this, mypy crashes)